src/hooks/useAuth.ts

```diff
@@ -67,8 +67,27 @@ export function useAuth() {
       });
   }
 
+  async function socialSignIn(type: string) {
```

Comment:
optional: type could be 'facebook' | 'google'

Reply:
ye ok y not